core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.constants.ts

```diff
@@ -16,7 +16,7 @@
  * @fileoverview Constants for the creator dashboard page.
  */
 
-interface TabDetails {
+export interface TabDetails {
```

Comment:
Check comment above. You can just get rid of this of this type.

Reply:
@JeeveshGarg Could you please make sure to address all the old comments? This is another one that has not been addressed.

Reply:
@JeeveshGarg Not sure what "above" means (in your comment). Could you please point to it or repeat the comment here too? It is hard to navigate a large PR so probably better to avoid too much referencing (as codeowners see only a subset of files).

Reply:
Done, removed